public/js/user-actions.js

```diff
@@ -13,13 +16,56 @@ function UserActions(localDispatcher) {
 }
 
 
-UserActions.prototype.setCurrentUser = function(user) {
-  console.log('UserActions: setting current user', user);
-  this.dispatcher.dispatch({
-    actionType: 'set-user',
-    user: user,
-  });
-};
+UserActions.prototype = assign({}, {
+
+  signIn: function(accessToken, opt) {
+    opt = opt || {};
+    if (!opt.jquery) {
+      opt.jquery = $;
+    }
```

Comment:
It is required but we aren't mocking out require statements currently. I agree, it would be nicer to reload the module with mock requires but we haven't set that up yet.